test/e2e/utils.go

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/golang/glog"
+	"golang.org/x/oauth2/google"
+	cloudresourcemanager "google.golang.org/api/cloudresourcemanager/v1"
+	boskosclient "k8s.io/test-infra/boskos/client"
+	remote "sigs.k8s.io/gcp-compute-persistent-disk-csi-driver/test/binremote/binremote"
+)
+
+var (
+	boskos = boskosclient.NewClient(os.Getenv("JOB_NAME"), "http://boskos")
+)
+
+func setupInstanceAndDriver(instanceProject, instanceZone, instanceName, port, instanceServiceAccount string) (*remote.InstanceInfo, error) {
+	// Create the instance in the requisite zone
+	instance, err := remote.CreateInstanceInfo(instanceProject, instanceZone, instanceName)
+	if err != nil {
+		return nil, err
+	}
+
+	err = instance.CreateInstance(instanceServiceAccount)
+
+	if err != nil {
+		return nil, err
+	}
+
+	// Create Driver Archive
+	archiveName := "e2e_gce_pd_test.tar.gz"
```

Comment:
can you make this configurable

Reply:
this is the configuration (?). I don't think the person running the test needs to worry about setting the name for the tarball that gets uploaded. In fact the argument could be made that this should just be not configurable at all and just called `thepackage.tar.gz` or something